lib/fog/test_helpers/minitest/assertions.rb

```diff
@@ -0,0 +1,35 @@
+require "fog/schema/data_validator"
+
+# Format related hackery
+# allows both true.is_a?(Fog::Boolean) and false.is_a?(Fog::Boolean)
+# allows both nil.is_a?(Fog::Nullable::String) and ''.is_a?(Fog::Nullable::String)
+module Fog
```

Comment:
Yeah I've been thinking about that too. I thought the code in Shindo to ultimately go away. Meanwhile that's going to take time so better do it. I'm creating a file for the common code.